apps/console/src/features/organizations/components/organization-switch/organization-switch-breadcrumb.tsx

```diff
@@ -178,12 +217,83 @@ export const OrganizationSwitchBreadcrumb: FunctionComponent<OrganizationSwitchD
             );
 
             mutateOrganizationBreadCrumbFetchRequest();
-            history.push(AppConstants.getPaths().get("GETTING_STARTED"));
+            redirectToStart ? history.push(AppConstants.getPaths().get("GETTING_STARTED")) : null;
```

Comment:
Do we need a ternary here.

```
mutateOrganizationBreadCrumbFetchRequest();

if (redirectToStart) {
     history.push(AppConstants.getPaths().get("GETTING_STARTED"))
}

Reply:
Fixed with 73a571a51f3bb8468e5cb6aeb5b66b5ea27ffee7